osu.Game.Rulesets.Osu/Edit/Setup/OsuSetupSection.cs

```diff
@@ -49,6 +49,7 @@ protected override void LoadComplete()
         private void updateBeatmap()
         {
             Beatmap.BeatmapInfo.StackLeniency = stackLeniency.Current.Value;
+            Beatmap.UpdateAllHitObjects();
```

Comment:
This bypasses the following guard:

https://github.com/ppy/osu/blob/a8a2e543d51788b1dc05ab318b54b46af76d0be7/osu.Game/Screens/Edit/EditorBeatmap.cs#L349-L350

Compare:

https://github.com/ppy/osu/blob/a8a2e543d51788b1dc05ab318b54b46af76d0be7/osu.Game/Screens/Edit/Setup/DifficultySection.cs#L119-L132